types/tampermonkey/index.d.ts

```diff
@@ -144,12 +134,7 @@ declare namespace Tampermonkey {
          * - `not_succeeded` - the download wasn't started or failed, the
          * details attribute may provide more information
          */
-        error:
-            | 'not_enabled'
-            | 'not_whitelisted'
-            | 'not_permitted'
-            | 'not_supported'
-            | 'not_succeeded';
+        error: 'not_enabled' | 'not_whitelisted' | 'not_permitted' | 'not_supported' | 'not_succeeded';
```

Comment:
Is this an intentional change or Prettier? Please disable any formatters for the code submitting here

Reply:
I just used prettier with the default settings but I can undo all these changes otherwise.

Reply:
I am on the fence about using formatters for type definitions. 
I mean we don't have `.prettierrc` in the root and everybody could have a different prettier configuration and even a different formatter.
If we want to apply auto formatting it should be a separate PR